### PR TITLE
feat(expressive): updated the chevron icon in timepicker as per dropdown

### PR DESCRIPTION
**Pull request by mkothur**
### Related Ticket(s)

Expressive theme - #1085

### Description
Updated the chevron icon in time picker as per the drop-down component.
image
image

### Changelog

**Changed**

  height of chevron inside the select of time-picker.

### CODE REVIEW

File: packages/styles/scss/themes/expressive/components/_time-picker.scss [hunk 1]
```diff
@@ -19,4 +19,9 @@
     height: carbon--rem(48px);
     width: carbon--rem(85px);
   }
+  .#{$prefix}--time-picker__select {
+    svg {
+      height: 100%;
```
Comment (annawen1) [inline packages/styles/scss/themes/expressive/components/_time-picker.scss]: Not sure if setting the `height` changes anything

I think what @oliviaflory was trying to say in the ticket - if you compare the dropdown chevron and the time picker chevron, the time picker chevron looks bigger


Screen Shot 2020-01-17 at 11 36 23 AM
Screen Shot 2020-01-17 at 11 36 39 AM

Comment (mkothur) [reply packages/styles/scss/themes/expressive/components/_time-picker.scss]: Okay got it

Comment (mkothur) [reply packages/styles/scss/themes/expressive/components/_time-picker.scss]: @annawen1: after looking into this, the `Dropdown` component is using the lighter version of the chevron icon. And time picker is using a darker version icon. Where these icons are being referenced in the themes.

**Comment by annawen1** [comment]
Don't think this PR is necessary anymore as we are switching the icon to use the same one as the `<Select />` component which already has expressive styles applied to it. 

PR submitted to Carbon: https://github.com/carbon-design-system/carbon/pull/5115

**Pull request closed without merging**